docs/notes/notes.rst

```diff
@@ -1,20 +1,21 @@
 .. _release_notes:
 
-Version 2.1.3
+Version 2.1.4
 =============
 
 This release includes the following **improvements**:
 
-1. Improve rediscovery on lossy environments.
+1. Improve auto gaps in data sharing
+2. Improve behavior when HAVE_STRICT_REALTIME is not set
```

Comment:
This are backports and Fast DDS v2.10.0 release notes already include them. I suggest unifying the notes and use the same as in that release.